src/main/scala/mesosphere/marathon/core/task/tracker/InstanceTrackerModule.scala

```diff
@@ -28,11 +28,13 @@ class InstanceTrackerModule(
   lazy val instanceTrackerUpdateStepProcessor: InstanceTrackerUpdateStepProcessor =
     new InstanceTrackerUpdateStepProcessorImpl(metrics, updateSteps)
 
-  private[this] lazy val updateOpResolver: InstanceUpdateOpResolver = new InstanceUpdateOpResolver(clock)
```

Comment:
This is a cleanup; InstanceTrackerActor.props is public but accepted things that could not be publicly instantiated. It's been rewritten to accept only public things now, and do the interim construction of these private objects.

The test has been updated to use the actor constructor directly in order to pass mocks and stubs of these things in.